website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -1,3 +1,78 @@
 ## Usage
 
-Usage documentation for this component is coming soon. In the meantime, help us improve this documentation by letting us know how your team is using it in [#team-design-systems](https://hashicorp.slack.com/archives/C7KTUHNUS) (internal only).
\ No newline at end of file
+### When to use
+
+- To navigate to a new destination, either internally or externally.
+- Wihtin a block of body text or other text element.
+
+### When not to use
+
+- As a standalone element, consider [Standalone Link](/components/link/standalone)
+- To trigger an action or event, like a form submission, consider [Button](/components/button)
+
+## Usage in Figma
+
+The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
+
+- Text styles can be found under text styles `Body` or `Code`.
```

Comment:
Technically (and practically) also headings can have links, no? (even if rare)
So should we include also `Display` here?

Reply:
That's a good point, also probably warrants a bit of additional guidance around when that might be appropriate. In my view, ideally never, but I know I've seen it.

Reply:
atm, we don't support this as a style in Figma though, which is what this section is talking about.

Making an actual change to offer it as a style in Figma is outside the scope of this ticket and warrants a larger discussion about whether or not this is something we want to support.